packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/class/punctuationChars.st

```diff
@@ -0,0 +1,5 @@
+constants
+punctuationChars
```

Comment:
Should move to instance side (as it is only used there atm).
Would also consider renaming it to `punctuationCharacters`.
Same for `whitespaceChars`.